sdk/elastic/Azure.ResourceManager.Elastic/samples/Generated/Samples/Sample_ElasticMonitorResource.cs

```diff
@@ -377,7 +377,7 @@ public async Task UpgradeMonitor_MonitorUpgrade()
         public async Task GetAllTrafficFilter_AllTrafficFiltersList()
         {
             // Generated from example definition: specification/elastic/resource-manager/Microsoft.Elastic/stable/2024-03-01/examples/AllTrafficFilters_list.json
-            // this example is just showing the usage of "AllTrafficFilters_List" operation, for the dependent resources, they will have to be created separately.
+            // this example is just showing the usage of "AllTrafficFilters_list" operation, for the dependent resources, they will have to be created separately.
```

Comment:
it's `list`: https://github.com/Azure/azure-rest-api-specs/blob/49456eb149e3bb5d0528cbca714cfa1806415484/specification/dynatrace/resource-manager/Dynatrace.Observability/stable/2023-04-27/dynatrace.json#L100